filebeat/module/nginx/access/ingest/default.json

```diff
@@ -95,9 +95,10 @@
         {
             "date": {
                 "if": "ctx.event.timezone != null",
-                "field": "@timestamp",
+                "field": "nginx.access.time",
+                "target_field": "@timestamp",
                 "formats": [
-                    "ISO8601"
+                    "dd/MMM/yyyy:H:m:s Z"
```

Comment:
This time format already includes a timezone, so timezone conversion shouldn't be needed. Could you remove this date processor? It should be enough with the first one.

Reply:
ok